src/luascript.cpp

```diff
@@ -1163,6 +1164,13 @@ void LuaScriptInterface::registerFunctions()
 	luaL_register(luaState, "result", LuaScriptInterface::luaResultTable);
 	lua_pop(luaState, 1);
 
+	// logger
+	lua_register(luaState, "logD", LuaScriptInterface::luaLogDebug);
+	lua_register(luaState, "logI", LuaScriptInterface::luaLogInfo);
+	lua_register(luaState, "logW", LuaScriptInterface::luaLogWarning);
+	lua_register(luaState, "logE", LuaScriptInterface::luaLogError);
+	lua_register(luaState, "logF", LuaScriptInterface::luaLogFatal);
```

Comment:
It could be a `class` with some methods

```lua
Log.debug("debug")
Log.info("info")
Log.warn("warn")
Log.error("error")
Log.fatal("fatal")
```